### PR TITLE
session 3 assignment

### PR DESCRIPTION
**Pull request by perezavidien**

### CODE REVIEW

File: ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        }
+
+        [HttpGet]
+        int Get(int userId)
+        {
+            return 1;
+        }
+
+        [HttpPost]
+        void Save(ItemCreateBindingModel modelObject)
```
Comment (jegtugado) [inline ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs:39]: Hi maam, need to add `[FromBody]` tag for model binding to occur. Checking the model state for the validation result is also required unless we have the `[ApiController]` attribute which was discussed earlier during session 4

File: Services/ItemService.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        public IEnumerable<string> GetAll(int userId)
+        {
+            // no requirements on what to do on the userId
+            return Items;
+        }
+
+
+        public void Save(ItemDTO itemObject)
+        {
+            Console.Write(itemObject);
```
Comment (jegtugado) [inline Services/ItemService.cs:23]: Please map ItemDTO to an Item domain model here. You can use your constructor approach just create an instance of Item object

**Review by jegtugado — CHANGES_REQUESTED** [review]
Looks good just need minor adjustments 👍

**Review by jegtugado — APPROVED** [review]

**Comment by jegtugado** [comment]
Please resolve conflicts so we can merge. Thank you!

File: Services/ItemService.cs (hunk 2 above)
Comment (perezavidien) [reply Services/ItemService.cs]: this is updated

File: ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs (hunk 1 above)
Comment (perezavidien) [reply ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs]: this is updated

**Comment by perezavidien** [comment]
added 1 more push - resolved conflicts,

**Pull request merged**